### DISCUSSION TITLE
Snow Conditions & Forecasts

### DISCUSSION THREAD

**Discussion by stvnbryntksh** [weather-gov/api#663] [Q&A]
Which endpoint and properties are best for snow conditions & forecasts?

Thanks

**Comment by stvnbryntksh** [comment]
I see these snow/ice properties: 
**snowLevel** which I believe is the elevation at which snow will occur given there is precipitation (?)
**snowfallAmount** snow that will fall in a given time period
**iceAccumulation** does this represent accumulated snow on ground?

**Reply by JJKraw** [reply]
I believe both snowfallAmount and iceAccumulation are the amount of new precipitation of each type during the time period specified.

**Reply by stvnbryntksh** [reply]
Hi, thanks, yeah makes sense. I still have a few questions:

1) What is "ice" in this context — hail? 
2) Is there a measurement for current snow DEPTH on the ground?
3) If not, a proxy for current snow depth on the ground could be determined by adding actual snowfall readings for a window of time — can ACTUAL previous snow fall amounts IN THE PAST be queried?

**Reply by JJKraw** [reply]
> What is "ice" in this context — hail?

Ice accumulation from freezing rain.  More info here: https://media.cocorahs.org/docs/Ice_Accretion_Pilot.pdf

> Is there a measurement for current snow DEPTH on the ground?

Not in the forecast API. I don't recall if that shows up in the observations endpoint but even if it did, you don't have an observation station in most grid boxes.

> If not, a proxy for current snow depth on the ground could be determined by adding actual snowfall readings for a window of time — can ACTUAL previous snow fall amounts IN THE PAST be queried?

The number of "official" snowfall reporting stations is very limited (usually airports). If there are any readings for particular locations that are reported by the NWS (via Local Storm Report or Public Information Statement), they are done by volunteers, some of whom have training (SKYWARN), some who don't. Snowfall measurement is tricky due to compaction, drifting, and the sheer variation in amounts that can be observed short distances apart. I'm a trained spotter and sometimes I question myself on what I'm reporting because what I see in my back yard (where my "snow board" is) is different than the front yard.  So what you see in these reports is a best effort rather than anything official.

Probably the best source for snow _depth_ is CoCoRaHS. Volunteers (hopefully) report their depth measurements every morning. But again, this number, at least in my case, is an average of what I see on my lot. There can be a lot of variation in the amount of compaction, melting, and sublimation, which makes for a challenge in coming up with one number to report. It's not uncommon to have both bare spots and other places that have six inches.

**Reply by dwhitemv25** [reply]
If you are looking in mountainous areas, then [SNOTEL](https://www.nrcs.usda.gov/wps/portal/wcc/home/quicklinks/imap) sites may also be available, though their snow depth reporting can be a little dodgy depending on conditions. SNOTEL does maintain historical data.

**Reply by stvnbryntksh** [reply]
Makes sense. I really appreciated all of the information. Who knew weather could be so complicated? jk ;)
